src/app/AttributePersistenceAccessors.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <app/AttributePersistenceProvider.h>
+#include <app/util/attribute-storage.h>
+
+namespace chip {
+namespace app {
+namespace AttributePersistenceAccessors {
+
+CHIP_ERROR ReadUint8Value(const ConcreteAttributePath & aPath, uint8_t & aValue)
+{
+    uint8_t attrData[1];
+    MutableByteSpan tempVal(attrData, 1);
+    auto err = GetAttributePersistenceProvider()->ReadValue(aPath, ZCL_INT8U_ATTRIBUTE_TYPE, 1, tempVal);
```

Comment:
Please instead add methods to AttributePersistenceProvider base class, or pass in the AttributePersistenceProvider as an arg, so we break this global singleton dependency and cause the caller to be able to inject location of AttributePersistenceProvider